toolkit/docs/nvidia/nvidia.md

```diff
@@ -0,0 +1,19 @@
+# Nvidia Repository Configuration
+
+## Overview
+The following documentation describes how to access CBL-Mariner packages from the NVIDIA RPM repository at [packages.microsoft.com](https://packages.microsoft.com/cbl-mariner/1.0/prod/NVIDIA/)
+
+## Licensing
+The software in the NVIDIA RPM repository is subject to the following:
+
+> NVIDIA Software. The software may include components developed and owned by NVIDIA Corporation or its licensors. The use of these components is governed by the NVIDIA end user license agreement located at [https://www.nvidia.com/content/DriverDownload-March2009/licence.php?lang=us](https://www.nvidia.com/content/DriverDownload-March2009/licence.php?lang=us). 
+
+## Instructions
+The following instructions register the nvidia package store with the package manager.
```

Comment:
Adding the mentioned subpackage would do these steps for the user, so maybe we won't need this documentation.

Reply:
These manual steps are by design.  As mentioned last week, we are mirroring NVIDIA's licensing agreement and approach.   You must take a manual action to get this.